src/imaSgxExternalSigner.ts

```diff
@@ -145,15 +144,15 @@ async function run() {
             process.exit( 0 );
         } catch ( err ) {
             if( gIsDebugLogging )
-                log.debug( "--- Call error ---> {}", log.em,( err ) );
-            finalizeOutput( { "error": owaspUtils.extractErrorMessage( err ) } );
+                log.debug( "--- Call error ---> {}", log.em, ( err ) );
+            finalizeOutput( { error: owaspUtils.extractErrorMessage( err ) } );
             process.exit( 1 );
         }
     } catch ( err ) {
         if( gIsDebugLogging )
             log.error( "RPC call to SGX failed: {err}", err );
-        finalizeOutput( { "error": owaspUtils.extractErrorMessage( err ) } );
+        finalizeOutput( { error: owaspUtils.extractErrorMessage( err ) } );
         process.exit( 1 );
     }
 }
-run();
+run().then( function() {} ).catch( function() {} );
```

Comment:
What is the purpose of this?

Reply:
ESLint error fix - call errors must be handled!

Reply:
Need to add some error log into catch

Reply:
Not need to add extra error logs in many cases like `.then( function() {} ).catch( function() {} );` because all errors logged inside